libsolidity/interface/CompilerStack.h

```diff
@@ -130,6 +130,13 @@ class CompilerStack: boost::noncopyable
 	/// @returns true if a source object by the name already existed and was replaced.
 	bool addSource(std::string const& _name, std::string const& _content, bool _isLibrary = false);
 
+	/// Adds a response to an SMTLib2 query (identified by the hash of the query input).
+	void addSMTLib2Response(h256 const& _hash, std::string const& _response) { m_smtlib2Responses[_hash] = _response; }
+
+	/// @returns a list of unhandled queries to the SMT solver (has to be supplied in a second run
+	/// by calling @a addSMTLib2Response.
+	std::vector<std::string> const& unhandledSMTQueries() const { return m_unhandledSMTLib2Queries; }
```

Comment:
Probably these two should match, ie `unhandledSMTLib2Queries`.

Also, it is nice to pair these two, but at the moment the functions are ordered in this file roughly by the order one can call them. That means this particular one should move after `compile` (well, after analyze, but parse/analyze/compiler are kept together).

Reply:
I thought about this, too, but it is actually much more similar to `errors()` and that is also rather to the start.

Reply:
Hm, I'd say `errors()` is an oversight in that case :)